mne_bids/mne_bids.py

```diff
@@ -533,8 +606,16 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     config : str | None
         A path to the configuration file to use if the data is from a BTi
         system.
-    overwrite : bool
-        If the file already exists, whether to overwrite it.
+    overwrite : str, one of ('overwrite', 'error', 'clear')
+        How any existing files should be handled when new data is produced.
+        Defaults to 'overwrite'.
+        - 'overwrite': Any file that already exists will be overridden with new
+            data, except `participants.tsv` and `scans.tsv` which will have the
+            new data appended intelligently.
+        - 'error': If a file already exists, raise an error (TBD).
```

Comment:
TBD is not clear ...

Reply:
Oh whoops! I forgot to change this from what I had written in the issue